spec/factories/tasks.rb

```diff
@@ -24,6 +25,7 @@
   factory :task, class: 'Task' do
     title { Faker::Lorem.sentence }
     description { Faker::Lorem.paragraph }
+    complete { Faker::Boolean.boolean }
```

Comment:
add traits complete/incomplete

Reply:
random true/false values are bad practice

Reply:
meaning - they are 50% predictable, and within a test example it may fail